client/my-sites/plugins/not-available-error.jsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'components/main';
+import EmptyContent from 'components/empty-content';
+import DocumentHead from 'components/data/document-head';
+import SidebarNavigation from 'my-sites/sidebar-navigation';
+
+function NotAvailableError( { translate } ) {
+	return (
+			<Main>
+				<DocumentHead title={ translate( 'Plugins' ) } />
+				<SidebarNavigation />
+				<EmptyContent
+					title={ translate( 'Not Available' ) }
+					line= { translate( 'The page you requested could not be found' ) }
```

Comment:
Do you think this error message makes sense for the context it's being used in? Can we display a more adequate error message in that case? Note that introducing such screens will need some feedback from design folks, so we should ping some of them if you decide to keep it.

Personally I'd suggest trying to keep the same behavior for the sake of this PR, and introducing such a message in a separate PR - keeping PRs smaller is always a good idea.